app/imports/startup/server/publications.js

```diff
@@ -142,6 +142,11 @@ Meteor.publishComposite('applications.index', function(posts, page, status, sort
   };
 
 
+  // Lock Committee members to Paid Applications
+  if (Roles.getUsersInRole('committee')) {
```

Comment:
this is the wrong condition and returns and array thus true. Should be `userIsInRole`